temporian/implementation/numpy/data/display_utils.py

```diff
@@ -340,10 +343,31 @@ def display_html_header(dom: Dom, evset: EventSet) -> Html:
     return root
 
 
+def repr_value_html(value: Any, dtype: DType) -> str:
```

Comment:
Any reason this block was moved?

Reply:
I moved the text related methods together, below the html ones. They were all over the place.